docs/README.md

```diff
@@ -0,0 +1,115 @@
+
+* Decide how you'll connect your receiver by reading [this document](https://github.com/cleanflight/cleanflight/blob/master/docs/Rx.md), and how many pins you need on the outputs (to connect ESCs and servos) by reading about [Mixers](https://github.com/cleanflight/cleanflight/blob/master/docs/Mixer.md). 
+
+* If you're interested in monitoring your flight battery with CF, see [Battery Monitoring](https://github.com/cleanflight/cleanflight/blob/master/docs/Battery.md).
+
+* You may want audible feedback from your copter so skim through [Buzzer](https://github.com/cleanflight/cleanflight/blob/master/docs/Buzzer.md) and mark the pins that will be used.
+
+* Do you want your RC Receiver's RSSI to be sent to the board? [This](https://github.com/cleanflight/cleanflight/blob/master/docs/Rssi.md) explains how. You may or may not need to make an additional connection from your Receiver to the FC.
+* Would you like to try using a GPS unit to get your aircraft to Loiter or Return-To-Launch? Take a look [here](https://github.com/cleanflight/cleanflight/blob/master/docs/Gps.md) and [here](https://github.com/cleanflight/cleanflight/blob/master/docs/Gps_-_Tested_Hardware.md).
+
+* Now that you know what features you are going to use, and which pins you need, you can go ahead and solder them to your board, if they are not soldered already. Soldering only the pins required for the application may save weight and contribute to a neater looking setup, but if you need to use a new feature later you may have to unmount the board from the craft and solder missing pins, so plan accordingly.
+
+* If you are going to use [Oneshot125](https://github.com/cleanflight/cleanflight/blob/master/docs/Oneshot.md), flash your ESCs with the latest stable firmware and enable Damped Light in their settings, if it's supported.
+
+## Software setup
+Now that your board has pins on it, you are ready to connect it to your PC and flash it with CF. Install the Chromium browser or Google Chrome to your PC, if you don't have it already, add the [CleanFlight Configurator]() to it, and start it.
+
+Then follow these instructions for [Installation](https://github.com/cleanflight/cleanflight/blob/master/docs/Installation.md) of the firmware to the FC.
+
+## CleanFlight Configuration
+Your FC should now be running CF, and you should be able to connect to it using the Configurator. If that is not the case, please go back to the previous sections and follow the steps carefully.
+
+<!--- This next paragraph should probably contain less info, as this info already exists in Configuration.md -->
+Now, there are two ways to [configure CF](https://github.com/cleanflight/cleanflight/blob/master/docs/Configuration.md); via  the Configurator's tabs (in a "graphical" way, clicking through and selecting/changing values and tickboxes) and using the [Command Line Interface (CLI)](https://github.com/cleanflight/cleanflight/blob/master/docs/Cli.md). Some settings may only be configurable using the CLI.
```

Comment:
... and some settings are best configured using the GUI (particularly the ports settings, which aren't documented for the CLI as they're not human friendly).